rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.create/21_synthetic_source_stored.yml

```diff
@@ -0,0 +1,732 @@
+---
+object param - object array:
```

Comment:
Are these just moved yaml tests?

Reply:
Yeah moved the ones with `store_array_source` and added some similar with the index setting.